components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -56,7 +57,7 @@ def create_parser():
 
 def main(argv=None):
   parser = create_parser()
-  args = parser.parse_args()
+  args = parser.parse_args(argv)
```

Comment:
why do we need this change with line 77 ?

Reply:
in main, parse_args() automatically takes argv input if we don't specify anything.
But when we mock the function it doesn’t.

So we have to specify the parameter